editors/vscode/package.json

```
@@ -94,6 +94,10 @@
                     },
                     "description": "The command line arguments passed to the Slint LSP server"
                 },
+                "slint.preview.show_ui": {
+                    "type": "boolean",
+                    "description": "Show the preview UI around the actual preview"
```

Needs a better description. Maybe something like:

```suggestion
                    "description": "Show the preview toolbar"
```

---

But "Preview UI" is not the right term as for most people that's the actual preview.

And we can change that once we have more UI.

---

I agree with Olivier. "preview UI around the actual preview" is weird and too much "preview". Toolbar is a better term that people may find themselves familiar with.